manifests/windows_service.pp

```diff
@@ -52,7 +53,7 @@
   } ->
   exec { 'consul_service_set_parameters':
     cwd         => $consul::bin_dir,
-    command     => "${consul::bin_dir}/set_service_parameters.ps1",
+    command     => "& \"${consul::bin_dir}/set_service_parameters.ps1\"",
```

Comment:
The ampersand (the 'and' sign) here tells PowerShell to execute that command, instead of treating it as a cmdlet or a string. The backticks (the funny looking single-quotes) are there to escape the following character, similar to the \" in C-based languages, or double-double-quotes ("") in VB. Otherwise the " character will end the string and the parser will cry when it can't understand what you're trying to say after that. (You can alternatively use single-quotes instead in this case, as I have in the previous example.)
http://edgylogic.com/blog/powershell-and-external-commands-done-right/
Just want to ensure this is executed as a command inside powershell, this is the way to force it to do so